cityofaustin/atd-moped
# Fix footer version link

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/8498

## Testing
**URL to test:** 
https://deploy-preview-602--atd-moped-main.netlify.app/moped/projects

**Steps to test:**
1. Click on the link in the footer. It should take you to the most recent Github Release Notes regardless of the version number listed.
2. Notice the change in the spacing and sizing of the External Link icon. It should align better with the text to the left of the icon.

---
#### Ship list
